files/build_templates/manifest.json.j2

```diff
@@ -0,0 +1,27 @@
+{
+    "version": "1.0.0",
+    "package": {
+        "version": "{{ version }}",
+        "base-os-contraint": "^{{ base_os_ver.split('.')[0] }}.0.0",
```

Comment:
Why only taking first octet from base_os_ver ?
What does the initial caret symbol imply?

Reply:
^1.1.0 is equivalent to ">=1.1.0 && <2.0.0".
It is not needed to limit the compatibility of package on every minor/patch version increment.
If base_os_ver is 1.5.6, ^1.5.6 will mean that this package can only work with 1.5.6 and upper but less then 2.0.0.'
So I am taking only major number.

Reply:
ok

Reply:
If you set constraint to ^1.5.6, 1.5.5 won't satisfy this constraint.
This is why I put only a constraint on major one, so e.g 1.0.0, 1.5.5, 1.9.9 - all satisfy this constraint.